### ISSUE TITLE
tropical cyclone TH

### ISSUE THREAD

**Issue by david-i-berry** [wmo-im/wis2-topic-hierarchy#161]
Noting the BUFR sequence under validation for tropical cyclones (https://github.com/wmo-im/BUFR4/issues/201) we do not have a WIS2 topic. To start the conversation I'd like to propose:

```
./weather/prediction/forecast/specialised-products/deterministic/tropical-storm-tracks
./weather/prediction/forecast/specialised-products/probabilistic/tropical-storm-tracks
```

cc [@sebvi](https://github.com/sebvi) for info.

Previous discussion here: https://github.com/wmo-im/tt-nwpmd/issues/12

**Comment by amilan17** [comment]
@sebvi @david-i-berry - this is the proposal from TT-NWPMD: 
Add trajectory topic as a sister to global and limited-area topics at the last level. 

- /weather/prediction/forecast/medium-range/deterministic/**trajectory**
- /weather/prediction/forecast/medium-range/probablistic/**trajectory**
- /weather/prediction/forecast/short-range/deterministic/**trajectory**
- /weather/prediction/forecast/short-range/probablistic/**trajectory**

**Comment by amilan17** [comment]
update branch and create PR for FT2024-2

**Comment by amilan17** [comment]
@wmo-im/tt-nwpmd  -- Trajectory topic needs a definition.  
@sebvi @yhonda21 

Name,Description,Source
global,A prediction system that covers the entire globe,
limited-area,A prediction system that does not cover the entire globe,
trajectory,**tbd**,

**Comment by amilan17** [comment]
TT-NWPMD meeting notes: 
The team likes using the IMV vocabulary but has some suggestions for revising the definition slightly to be more generic. @amilan17 to find out if it's possible to update IMV.

**Comment by amilan17** [comment]
@amilan17 update description and open PR

**Comment by amilan17** [comment]
combine 1 & 2 -- 
- Line in space tracing the successive positions of a moving particle in the air or the centre of a weather system. 
- Eunha will double check with english editor.

**Comment by amilan17** [comment]
@EunhaLIM @sebvi I updated the branch with the following definition. I decided to not use the air mass one. 

trajectory,"Line in space tracing the successive positions of (i) moving particles in the air, such as radioactive particles from a nuclear accident or (ii) the centre of a weather system, such as a tropical cyclone.",

**Comment by amilan17** [comment]
CORRECTION: 
@EunhaLIM @sebvi I updated the branch with the following definition. I decided to not use the air mass one, because I think "positions" could be represented as an air mass. I also changed "tracing" to "representing" because this is under the prediction topic.

trajectory,"Line in space representing the successive positions of (i) moving particles in the air, such as radioactive particles from a nuclear accident or (ii) the centre of a weather system, such as a tropical cyclone.",
